CHANGELOG.md

```diff
@@ -10,6 +10,37 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
 
 ### Release notes 
 
+* Features
+
+* Bug Fixes 
+  * Fix for preventing [cilium-operator](https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium)  from restarting unmanaged coredns pods
```

Comment:
According to the doc best practice, better to remove locale "en-us" from the URL, also apply to other doc linkage.